src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonSynchronisationService.kt

```diff
@@ -47,6 +48,8 @@ import java.time.LocalDate
 import java.time.LocalDateTime
 import java.time.Period
 
+private fun telemetryOf(vararg pairs: Pair<String, Any>): MutableMap<String, Any> = mutableMapOf(*pairs)
```

Comment:
reduce the noise for the types e.g <String, Any> everywhere can be removed

Reply:
Prob worth adding this to the Telemetry class so everyone can use it?